extension/content.js

```diff
@@ -220,6 +220,43 @@ function showRecentlyPushedBranches() {
 	$('.repository-content').prepend(div);
 }
 
+// Add option for viewing diffs without whitespace changes
+function addDiffViewWithoutWhitespaceOption(type) {
+	if ($('.diff-options-content').length < 1 && $('.btn-group .selected[href*="diff="]').length < 1) {
+		return;
+	}
+
+	const optionElement = document.createElement('a');
+	const urlParams = new URLSearchParams(window.location.search);
+	const urlArgs = window.location.search === '' ? '?w=1' : `${window.location.search}&w=1`;
+	const urlHash = window.location.hash || '';
+	const svgIcon = '<svg aria-hidden="true" class="octicon octicon-check" height="16" version="1.1" viewBox="0 0 12 16" width="12"><path d="M12 5L4 13 0 9l1.5-1.5 2.5 2.5 6.5-6.5 1.5 1.5z"></path></svg>';
+	const optionElementObject = $(optionElement);
+	let optionIsSet = false;
+	let optionHref = window.location.origin + window.location.pathname + urlArgs + urlHash;
```

Comment:
Does this need to be here? It looks like `optionHref` is always re-assigned below, without this value ever being used. Should be able to get rid of this line and make `optionHref` a const assignment